src/components/Datagrid/usePortal.ts

```diff
@@ -0,0 +1,13 @@
+import { useEffect } from "react";
+
+/**
+ * Glide automatically refers to the div with an id "portal"
+ * This hook allows to add additional classes for it
+ */
+const portalRoot = document.getElementById("portal");
+
+export const usePortal = ({ className }: { className: string }) => {
```

Comment:
If it's used just for styling can we name usePortalClasses or something like this